core/src/main/java/org/infinispan/manager/ClusterExecutionPolicy.java

```diff
@@ -0,0 +1,90 @@
+package org.infinispan.manager;
+
+import org.infinispan.remoting.transport.TopologyAwareAddress;
+
+/**
+ * ClusterExecutionPolicy controls how which nodes commands submitted via {@link ClusterExecutor}. That is the user
+ * can ensure that a command goes or doesn't go to a specific physical location such as on the existing machine, rack
+ * or site.
+ * <p>
+ * ClusterExecutionPolicy effectively scopes execution of commands to a subset of nodes. For
+ * example, someone might want to exclusively execute commands on a local network site instead of a
+ * backup remote network centre as well. Others might, for example, use only a dedicated subset of a
+ * certain Infinispan rack nodes for specific task execution.
+ *
+ *
+ * @author William Burns
+ * @since 9.0
+ */
+public enum ClusterExecutionPolicy {
+   /**
+    * The command can be executed on any node in the cluster
+    */
+   ALL {
+      @Override
+      public boolean include(TopologyAwareAddress thisAddress, TopologyAwareAddress otherAddress) {
+         return true;
+      }
+   },
+   /**
+    * The command can be executed only on the same machine from where the command was executed.  Note this implies
```

Comment:
Cacophonic. Better use something like:
The command can be executed only on the same machine from where it is initiated